Dockerfile

```diff
@@ -1,18 +1,23 @@
 FROM alpine:edge
 
+ENV K8S_VERSION=v1.6.3
```

Comment:
I'm fine with that. And quay should build this periodically, to get shiny new version of things.